### PR TITLE
Tests for Contract and MethodId

### PR DESCRIPTION
**Pull request by r8d8**

### CODE REVIEW

File: src/test/groovy/org/ethereumclassic/etherjar/model/MethodIdSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+        MethodId.from(invalid_string as String)
+        then:
+        thrown(IllegalArgumentException)
+        where:
+        _ | invalid_string
+        _ | null
+        _ | "123"
+        _ | "-850932852093457982375"
+    }
+}
```
Comment (dulanov) [inline src/test/groovy/org/ethereumclassic/etherjar/model/MethodIdSpec.groovy]: Please add also positive tests.

File: src/test/groovy/org/ethereumclassic/etherjar/contract/ContractSpec.groovy [hunk 2]
```diff
@@ -1,6 +1,61 @@
...
+        _ | valid_methods
+        _ | [new MethodType(new MethodId([0xff, 0x11, 0x22, 0x33] as byte[])),
+             new MethodType(new MethodId([0xff, 0x00, 0xCC, 0x44] as byte[])),
+             new MethodType(new MethodId([0xff, 0xAA, 0x2B, 0x3C] as byte[]))]
+        _ | []
+        _ | new ArrayList<MethodType>()
+        _ | new ArrayList<MethodType>() {{new MethodType(new MethodId([0xff, 0x11, 0x22, 0x33] as byte[]));
+                                            new MethodType(new MethodId([0xff, 0x11, 0x22, 0x33] as byte[]));
+                                            new MethodType(new MethodId([0xff, 0x11, 0x22, 0x33] as byte[]))}}
+    }
```
Comment (dulanov) [inline src/test/groovy/org/ethereumclassic/etherjar/contract/ContractSpec.groovy]: I think, two last variants with ArrayList are unnecessary, just wrapper.

File: src/test/groovy/org/ethereumclassic/etherjar/contract/ContractSpec.groovy [hunk 3]
```diff
@@ -1,6 +1,61 @@
...
+
+    def "check null method id"() {
+        when:
+        Contract contract = new Contract([new MethodType(new MethodId([0xff, 0x11, 0x22, 0x33] as byte[])),
+                                          new MethodType(new MethodId([0xff, 0x00, 0xCC, 0x44] as byte[]))])
+        contract.getMethod(null)
+        then:
+        thrown(IllegalArgumentException)
+
```
Comment (dulanov) [inline src/test/groovy/org/ethereumclassic/etherjar/contract/ContractSpec.groovy]: `null` must be inserted as data?

Comment (r8d8) [inline src/test/groovy/org/ethereumclassic/etherjar/contract/ContractSpec.groovy]: Nope, just test for null id request.

Comment (dulanov) [inline src/test/groovy/org/ethereumclassic/etherjar/contract/ContractSpec.groovy]: Just move `null` to where section for more clearness.

**Pull request merged**